src/domain/card.ts

```diff
@@ -0,0 +1,19 @@
+import { CARD_NUMBER } from '@/constants/card';
+
+export function isShowHyphen(prevCardNumber: string, nextCardNumber: string) {
+  return prevCardNumber.length === CARD_NUMBER.MAX_LENGTH && nextCardNumber.length > 0;
+}
+
+export function maskingNumber(cardNumber: string) {
+  return ''.padStart(cardNumber.length, '*');
+}
+
+export function isValidExpirationMonth(month: string) {
+  const [month1, month2] = month;
+
+  if (Number(month1) < 0 || Number(month1) > 1) return false;
+
+  if (Number(month1) === 1 && month2 !== undefined && ![0, 1, 2].includes(Number(month2))) return false;
+
+  return true;
+}
```

Comment:
```suggestion
export function isValidExpirationMonth(_month: string) {
  const 입력된자리수 = _month.length;
  const month = Number(_month);

  if (입력된자리수 === 0) return true;
  if (입력된자리수 === 1) return month < 2;

  return month !== 0 && month < 13;
}
```
조건문이 어떤것에 대한 검증을하는지 명확하게 파악하기가 힘든데 이렇게 바꿔보면 어떨까요? 👀

Reply:
훨씬 좋은 것 같습니다! 명확하게 한글로 변수명을 작성하고, 
작성된 조건들이 훨씬 읽기 쉽고 명확하게 작성이 되었네요. 
수의 범위에 관련된 조건문을 작성할 때 좋은 팁을 얻은 것 같습니다.
감사합니다!